CubeScramble.py

```diff
@@ -4,20 +4,22 @@
 # Example: R, Ri or R2
 
 
-
-
-import random
-
+from random import choice
 
 # all possible moves on a Rubik's Cube
 
-list = ['U', 'Ui', 'U2', 'F', 'Fi', 'F2', 'R', 'Ri', 'R2', 'B', 'Bi', 'B2', \
```

Comment:
Slashes are bad. Trailing white space can break them and it's super hard to see what when wrong. Never use them. There's always a better way.

Reply:
flake8 told me to take that out too. I think the code on my computer is more up to date. Let me push that to dirtyWork and we can work from there?

Reply:
Sure thing